Plan/common/src/main/java/com/djrapitops/plan/component/ComponentSvc.java

```diff
@@ -0,0 +1,96 @@
+/*
+ *  This file is part of Player Analytics (Plan).
+ *
+ *  Plan is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License v3 as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Plan is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with Plan. If not, see <https://www.gnu.org/licenses/>.
+ */
+package com.djrapitops.plan.component;
+
+import com.djrapitops.plan.exceptions.EnableException;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+/**
+ * Implementation for {@link ComponentService}.
+ *
+ * @author Vankka
+ */
+@Singleton
+public class ComponentSvc implements ComponentService {
+
+    private final ComponentConverter converter;
+
+    @Inject
+    public ComponentSvc() {
+        try {
+            Class<?> clazz = Class.forName("com.djrapitops.plan.component.ComponentConverterImpl");
+            this.converter = (ComponentConverter) clazz.getDeclaredConstructor().newInstance();
+        } catch (ReflectiveOperationException e) {
+            throw new EnableException("Could not initialize ComponentConverter", e);
```

Comment:
Throwing EnableException during class initialization is not caught by the exception handler https://github.com/plan-player-analytics/Plan/blob/master/Plan/bukkit/src/main/java/com/djrapitops/plan/Plan.java#L71